packages/commcare/src/Adaptor.js

```diff
@@ -68,7 +68,11 @@ export function execute(...operations) {
  * @param {function} [callback] - Optional callback to handle the response
  * @returns {Operation}
  */
-export function get(path, params = {}, callback = s => s) {
+export function get(
+  path,
+  params = { offset: 0, limit: 1000 },
```

Comment:
I'm not sure these defaults are set the correct way.

If the user does `get('blah', { offset: 10 })`, then limit won't be defaulted.

I think after we expand the params, we should deconstruct and default them.

We also need to change the params passed into the request to ensure it uses the deconstructed values